browser-test/src/admin_program_creation.test.ts

```diff
@@ -19,6 +20,26 @@ describe('program creation', () => {
     await validateScreenshot(page, 'program-description-page')
   })
 
+  it('View active program, without draft and after creating draft', async () => {
+    const {page, adminPrograms} = ctx
+
+    await loginAsAdmin(page)
+    await enableFeatureFlag(page, 'program_read_only_view_enabled')
+
+    const programName = 'Apc program'
+    await adminPrograms.addProgram(programName)
+    await adminPrograms.gotoAdminProgramsPage()
+
+    await adminPrograms.publishAllPrograms()
+    await validateScreenshot(page, 'program-list-only-one-active-program')
+    await adminPrograms.viewActiveVersion(programName)
+    // TODO(jhummel) add screenshot here when the other pull request is submitted
+    // await validateScreenshot(page, 'program-read-only-viewer')
+    await adminPrograms.createNewVersionWithReadOnlyViewEnabled(programName)
+    await validateScreenshot(page, 'program-list-active-and-draft-program')
```

Comment:
**issue (blocking):** Screenshots are great, but we should also be doing dom element checks to make the error messages more pointed towards issues.  See other tests in this file for example.